pages/web-literacy/WebLiteracyPage.jsx

```diff
@@ -17,11 +17,23 @@ var topicContent = weblitcontent.topics;
 var webLitSkillsContent = weblitcontent.webLitSkills;
 var makeLinksFromWebLitSkills = require("./MakeLinksFromWebLitSkills.jsx");
 
+function convertToRoute(str) {
+  str = str || "";
+  return str.toLowerCase().replace(" ", "-");
+}
+
+function parseRoute(route) {
+  route = route || "";
+  return route.split("-").map(function(item) {
+    return item.charAt(0).toUpperCase() + item.substr(1).toLowerCase();
+  }).join(" ");
+}
```

Comment:
I might update the names too. Outside of this file the context gets lost, so I should be more clear.

But yeah, moving them to utils is fine with me.